src/course-unit/course-xblock/xblock-messages/utils.test.js

```diff
@@ -0,0 +1,44 @@
+import { MESSAGE_ERROR_TYPES } from '../constants';
+import { getMessagesBlockType } from './utils';
+
+describe('xblock-messages utils', () => {
+  describe('getMessagesBlockType', () => {
+    test('returns "warning" when there are no error messages', () => {
```

Comment:
[nit]: please replace "test" with "it" in this file
```suggestion
    test('returns "warning" when there are no error messages', () => {
```